templates/banner/[VENDOR]/[MODULE]/view/adminhtml/web/template/content-type/[CONTENTTYPELC]/collage-centered/master.html

```diff
@@ -0,0 +1,61 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+
+<!--
+  [MASTER_TEMPLATE_COMMENT]
+-->
+
+<div attr="data.main.attributes" ko-style="data.main.style" css="data.main.css">
+    <a if="data.link.attributes().href" attr="data.link.attributes" ko-style="data.link.style" css="data.link.css">
```

Comment:
As far as I see, we have only `main` element in the configuration. I think we should remove all element that not containts `main`